### ISSUE TITLE
Unexpected attribute 'query' found in query criteria

### ISSUE THREAD

**Issue by jamiegalbreath** [MichaelSolati/geofirestore#37]
It doesnt like query: (ref) => ref.where('d.count', '==', 1) in the following: 
 
```
   geoQuery = geoFirestore.query({
      center: new firebase.firestore.GeoPoint(mapCenter.lat(), mapCenter.lng()),
      radius: radius,
      query: (ref) => ref.where('d.count', '==', 1)
    });
```

Has error: Unexpected attribute 'query' found in query criteria

Any thoughts?

**Comment by MichaelSolati** [comment]
Could we see the package.json? (Or a broken example, like a stackblitz link?)

**Comment by MichaelSolati** [comment]
Closing this for now, but if you get the chance to share a working (broken) example we can definitely reopen this!

**Comment by jamiegalbreath** [comment]
See example here it works when below is removed...
`query: (ref) => ref.where('d.count', '==', 1)`

[](https://jsfiddle.net/jamiegalbreath/f5063ya2/8/)

**Comment by MichaelSolati** [comment]
@jamiegalbreath I finally got to test it, I see that the JSFiddle was using version `2.0.2`, however custom queries weren't available until v `2.1.0`. Try updating to the latest version `2.2.1` and you should be all set.

[You can see that here](https://jsfiddle.net/yuhL257j/), however it's not indexed 😅
